src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
`size_t.sizeof` is just a consequence of `D_LP64` version. For me it's a bad style to use such huge expressions like `static if (size_t.sizeof != ulong.sizeof)` and also create a third `else static assert(0);` case. I just don't want to refactor all the module.